src/log/service.ts

```diff
@@ -2,6 +2,8 @@ import { Context } from '@curveball/core';
 import db from '../database';
 import { User } from '../user/types';
 import { EventType, LogEntry } from './types';
+// @ts-ignore: No types for this yet
```

Comment:
It looks like there is a `@types/geoip-lite` package, so this should not be needed when this is installed!